src/builder-manifest.c

```diff
@@ -1090,6 +1100,16 @@ builder_manifest_class_init (BuilderManifestClass *klass)
                                                      G_MAXINT32,
                                                      -1,
                                                      G_PARAM_READWRITE));
+
+  g_object_class_install_property (object_class,
+                                   PROP_SOURCE_DATE_EPOCH,
+                                   g_param_spec_int64 ("source-date-epoch",
+                                                     "",
+                                                     "",
+                                                     0,
+                                                     G_MAXINT64,
+                                                     0,
+                                                     G_PARAM_READWRITE));
```

Comment:
```suggestion
                                                      "",
                                                      "",
                                                      0,
                                                      G_MAXINT64,
                                                      0,
                                                      G_PARAM_READWRITE));
```

Reply:
Hard to tell on the github UI, but alignment was just off by one or two.

Reply:
It was off by one more space sorry :/

Reply:
No problems, I should have been more careful in the first place.